src/main/frontend/src/components/ReviewTable.js

```diff
@@ -0,0 +1,173 @@
+    Check: forwardRef((props, ref) => <Check {...props} ref={ref}/>),
+    Clear: forwardRef((props, ref) => <Clear {...props} ref={ref}/>),
+    Delete: forwardRef((props, ref) => <DeleteOutline {...props} ref={ref}/>),
+    DetailPanel: forwardRef((props, ref) => <ChevronRight {...props} ref={ref}/>),
+    Edit: forwardRef((props, ref) => <Edit {...props} ref={ref}/>),
+    Export: forwardRef((props, ref) => <SaveAlt {...props} ref={ref}/>),
+    Filter: forwardRef((props, ref) => <FilterList {...props} ref={ref}/>),
+    FirstPage: forwardRef((props, ref) => <FirstPage {...props} ref={ref}/>),
+    LastPage: forwardRef((props, ref) => <LastPage {...props} ref={ref}/>),
+    NextPage: forwardRef((props, ref) => <ChevronRight {...props} ref={ref}/>),
+    PreviousPage: forwardRef((props, ref) => <ChevronLeft {...props} ref={ref}/>),
+    ResetSearch: forwardRef((props, ref) => <Clear {...props} ref={ref}/>),
+    Search: forwardRef((props, ref) => <Search {...props} ref={ref}/>),
+    SortArrow: forwardRef((props, ref) => <ArrowDownward {...props} ref={ref}/>),
+    ThirdStateCheck: forwardRef((props, ref) => <Remove {...props} ref={ref}/>),
+    ViewColumn: forwardRef((props, ref) => <ViewColumn {...props} ref={ref}/>)
+}
+
+const ReviewTable = () => {
+
+    const [reviewsData, setReviewsData] = useState([])
+    const [isError, setIsError] = useState(false)
+    const [errorMessages, setErrorMessages] = useState([])
+
+    useEffect(() => {
+        fetch("/api/v1/reviews")
+            .then((response) => {
+                if (response.ok) {
+                    return response
+                }
+            })
+            .then(response => response.json())
+            .then(reviews => {
+                console.log
+                let foodChainsArray = reviews.map((review) => {
+                    return {id: review.id, comment: review.comment, rating: review.rating, foodChain: review.foodChain.name}
+                }
+
+                );
+                setReviewsData(foodChainsArray)
+                console.log(foodChainsArray)
+            })
+            .catch(error => console.log(error))
+    }, [])
+
+    const columns = [
+        {title: "id", field: "id", hidden: true},
+        {title: "Food Chain", field: "foodChain"},
+        {title: "Rating", field: "rating"},
+        {title: "Comment", field: "comment"}
+    ]
+
+    const handleRowUpdate = (newData, oldData, resolve) => {
+        let errorList = []
+        if (newData.foodChain === undefined) {
+            errorList.push("Please make sure a food chain is selected for this review")
+        }
+
+        if (newData.rating === undefined) {
+            errorList.push("Please make sure review contains a rating")
+        }
+
+        if (newData.comment === undefined) {
+            errorList.push("Please make sure review contains a comment")
+        }
+
+        if (errorList.length == 0) {
+            fetch('/api/v1/reviews/' + newData.id, {
```

Comment:
It would be far more readable to string interpolate rather than concatenate here